### PR TITLE
feat: legal_entities review by nhs

### PR DESCRIPTION
**Pull request by svetavedmed**
https://github.com/edenlabllc/ehealth.api/issues/3469

### CODE REVIEW

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 1]
```diff
@@ -5,42 +5,32 @@ Feature: Test Legal Entity Details Page in NHS admin panel
...
   Then I should be logged in
   And I navigate to the LEGAL_ENTITY_DETAILS_PAGE
 
 Scenario: Get LEGAL_ENTITY Details
   Given I am on the LEGAL_ENTITY_DETAILS_PAGE
   Then I should see header with id and status
-  And I see tabs GENERAL_INFO, ACCREDETATION, RELATED_LEGAL_ENTITES, OWNER, DIVISIONS, CONTRACT_REQUESTS, CONTRACTS
-  And if status ='ACTIVE' 
+  And I see tabs GENERAL_INFO, ACCREDETATION, RELATED_LEGAL_ENTITES, OWNER, DIVISIONS, CONTRACT_REQUESTS, CONTRACTS, ACTIONS
+  And if status ='ACTIVE' and nhsReview=True
```
Comment (kiurchv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]: Let's write such things in a human-readable manner. 

```suggestion
  And if "status" is "ACTIVE" and "nhsReview" is true
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 2]
```diff
@@ -1,46 +1,47 @@
...
   Then I should be logged in
   And I navigate to the LEGAL_ENTITY_DETAILS_PAGE
 
 Scenario: Get LEGAL_ENTITY Details
   Given I am on the LEGAL_ENTITY_DETAILS_PAGE
   Then I should see header with id and status
-  And I see tabs GENERAL_INFO, ACCREDETATION, RELATED_LEGAL_ENTITES, OWNER, DIVISIONS, CONTRACT_REQUESTS, CONTRACTS
-  And if status ='ACTIVE' 
+  And I see tabs GENERAL_INFO, ACCREDETATION, RELATED_LEGAL_ENTITES, OWNER, DIVISIONS, CONTRACT_REQUESTS, CONTRACTS, ACTIONS
+  And if status ='ACTIVE' and nhsReview is True
```
Comment (ephor) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  And if status is ACTIVE and nhsReview is True
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 3]
```diff
@@ -1,46 +1,47 @@
...
+  And I approve scopes
   Then I should be logged in
   And I navigate to the LEGAL_ENTITY_DETAILS_PAGE
 
 Scenario: Get LEGAL_ENTITY Details
   Given I am on the LEGAL_ENTITY_DETAILS_PAGE
   Then I should see header with id and status
-  And I see tabs GENERAL_INFO, ACCREDETATION, RELATED_LEGAL_ENTITES, OWNER, DIVISIONS, CONTRACT_REQUESTS, CONTRACTS
-  And if status ='ACTIVE' 
+  And I see tabs GENERAL_INFO, ACCREDETATION, RELATED_LEGAL_ENTITES, OWNER, DIVISIONS, CONTRACT_REQUESTS, CONTRACTS, ACTIONS
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  And I see tabs GENERAL_INFO, LICENSES_ACCREDETATION, RELATED_LEGAL_ENTITES, OWNER, DIVISIONS
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 4]
```diff
@@ -1,46 +1,47 @@
...
+  And if status ='ACTIVE' and nhsReview is True
   Then I should see button DEACTIVATE
-  And if nhsVerified=false
-  Then I should see button VERIFY
 
 Scenario: DEACTIVATE LEGAL_ENTITY 
-  Given I am on the LEGAL_ENTITY_DETAILS_PAGE page
+  Given I am on the LEGAL_ENTITY_DETAILS_PAGE page and nhsReview is true
   When I press DEACTIVATE button
   Then I should see APPOVAL pop-up window
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  Then I should see APPROVAL pop-up window 
```

File: packages/admin/e2e/__features__/legal_entities_details.feature (hunk 3 above)
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  And I see tabs GENERAL_INFO, LICENSES_ACCREDETATION, RELATED_LEGAL_ENTITES, OWNER, DIVISIONS
```

File: packages/admin/e2e/__features__/legal_entities_details.feature (hunk 4 above)
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  Then I should see APPROVAL pop-up window 
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 5]
```diff
@@ -1,46 +1,47 @@
...
+  Then I should see LEGAL_ENTITY_DETAILS_PAGE tab 
   And status=CLOSED
 
-Scenario: NHS_VERIFY LEGAL_ENTITY 
-  Given I am on the LEGAL_ENTITY_DETAILS_PAGE page
-  When I press VERIFY button
+Scenario: NHS_REVIEW LEGAL_ENTITY 
+  Given I am on the LEGAL_ENTITY_DETAILS_PAGE page and nhsReview is false
+  When I press REVIEW button
   Then I should see APPOVAL pop-up window
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  Then I should see APPROVAL pop-up window 
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 6]
```diff
@@ -1,46 +1,47 @@
...
   And I press YES
-  Then I shouls see LEGAL_ENTITY_DETAILS_PAGE tab 
-  And nhsVerified=true
+  Then I should see LEGAL_ENTITY_DETAILS_PAGE tab
+  And I should see button DEACTIVATE
 
 Scenario: Get LEGAL_ENTITY Details (GENERAL_INFO)
   Given I am on the GENERAL_INFO tab
-  Then I should see type, name, edrpou, phones, email, addresses, kveds, ownerPropertyType,  misVerified, nhsVerified
+  Then I should see type, name, edrpou, phones, email, addresses, kveds, ownerPropertyType, website, receiver_funds_code, legal_form, beneficiary, archive_place, archive_date
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  Then I should see type, name, edrpou, phones, email, addresses, kveds, ownerPropertyType, website, receiverFundsCode, legalForm, beneficiary, archivePlace, archiveDate, misVerified
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 7]
```diff
@@ -77,5 +78,17 @@ Scenario: ADD_NEW_RELATED_LEGAL_ENTITY
...
+  Given I am on the ACCREDETATION_VERIFICATION page and nhsReview is true
+  When I press VERIFY button
+  Then I should see APPOVAL pop-up window 
+  And I press YES
+  Then I should see ACCREDETATION_VERIFICATION tab 
+  And UNVERIFY button
 
-
+Scenario: COMMENT LEGAL_ENTITY
+  Given I am on the ACCREDETATION_VERIFICATION tab and nhsReview is true
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  Given I am on the LICENSES_ACCREDITATION tab and nhsReview is true
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 8]
```diff
@@ -77,5 +78,17 @@ Scenario: ADD_NEW_RELATED_LEGAL_ENTITY
...
+  Then I write comment in a COMMENT form
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  Then I should see button ADD_COMMENT
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 9]
```diff
@@ -77,5 +78,17 @@ Scenario: ADD_NEW_RELATED_LEGAL_ENTITY
...
+  When I submit COMMENT form
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  And I press ADD_COMMENT button
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 10]
```diff
@@ -77,5 +78,17 @@ Scenario: ADD_NEW_RELATED_LEGAL_ENTITY
...
+  Then I should see ACCREDETATION/VERIFICATION tab
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  Then I return to same tab
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 11]
```diff
@@ -77,5 +78,17 @@ Scenario: ADD_NEW_RELATED_LEGAL_ENTITY
...
+  And COMMENT form with entered text
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  And I see typed text under the button ADD_COMMENT
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 12]
```diff
@@ -77,5 +78,17 @@ Scenario: ADD_NEW_RELATED_LEGAL_ENTITY
   And I see the message that request in status PENDING 
   And i'm returned to ASYNC_JOB page
 
+Scenario: NHS_VERIFY LEGAL_ENTITY 
+  Given I am on the ACCREDETATION_VERIFICATION page and nhsReview is true
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  Given I am on the LICENSES_ACCREDITATION page and nhsReview is true and nhsVerified is false
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 13]
```diff
@@ -77,5 +78,17 @@ Scenario: ADD_NEW_RELATED_LEGAL_ENTITY
...
+  When I press VERIFY button
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  Then I should see button VERIFY_LEGAL_ENTITY
  When I press VERIFY_LEGAL_ENTITY button
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 14]
```diff
@@ -77,5 +78,17 @@ Scenario: ADD_NEW_RELATED_LEGAL_ENTITY
...
+  Then I should see APPOVAL pop-up window
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  Then I should see APPROVAL pop-up window and buttons BACK and VERIFY_LEGAL_ENTITY 
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 15]
```diff
@@ -77,5 +78,17 @@ Scenario: ADD_NEW_RELATED_LEGAL_ENTITY
   And I see the message that request in status PENDING 
   And i'm returned to ASYNC_JOB page
 
+Scenario: NHS_VERIFY LEGAL_ENTITY 
+  Given I am on the ACCREDETATION_VERIFICATION page and nhsReview is true
+  When I press VERIFY button
+  Then I should see APPOVAL pop-up window 
+  And I press YES
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  And I press VERIFY_LEGAL_ENTITY
```

File: packages/admin/e2e/__features__/legal_entities_details.feature [hunk 16]
```diff
@@ -77,5 +78,17 @@ Scenario: ADD_NEW_RELATED_LEGAL_ENTITY
...
+  Then I should see ACCREDETATION_VERIFICATION tab
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_details.feature]:
```suggestion
  Then I should see LICENSES_ACCREDITATION tab 
```

File: packages/admin/e2e/__features__/legal_entities_list.feature [hunk 17]
```diff
@@ -11,22 +11,27 @@ Background: User is authorized and on DECLARATION_LIST_PAGE
 Scenario: Search for a Legal Entity using ID
   Given I fill in input search field with correct value
   And I submit form
-  Then I should see grid with name, edrpou, addresses, status, nhsVerified, insertedAt, details
-  And I should be able to order table by edrpou_ASC, edrpou_DESC, status_ASC, status_DESC, nhsVerified_ASC, nhsVerified_DESC, insertedAt_ASC, insertedAt_DESC
+  Then I should see grid with name, edrpou, addresses, status, nhsReview, nhsVerified, insertedAt, details
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_list.feature]:
```suggestion
  Then I should see table:
  | id  			  | name 	 | edrpou   | type 		  | addresses         | status | nhsReview | nhsVerified | insertedAt | action  |
  | 53XXX1d-XXXX-4fa3 | Lymych M | 31023234 | Description | Kyiv, Bazhana 12b | ACTIVE | true  	   | false 		 | 12.11.2018 | details |
  | 11XXX2X-XXXX-XXXX | Boris  M | 31323234 | Description | Kyiv, Bazhana 12b | ACTIVE | false 	   | true 		 | 12.11.2018 |	details |
```

File: packages/admin/e2e/__features__/legal_entities_list.feature [hunk 18]
```diff
@@ -11,22 +11,27 @@ Background: User is authorized and on DECLARATION_LIST_PAGE
...
+  And I should be able to order table by edrpou_ASC, edrpou_DESC, status_ASC, status_DESC, nhsReviewed_ASC, nhsReviewed_DESC, nhsVerified_ASC, nhsVerified_DESC, insertedAt_ASC, insertedAt_DESC
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_list.feature]:
```suggestion
  And I should be able to order table by EDRPOU_ASC, EDRPOU_DESC, STATUS_ASC, STATUS_DESC, NHS_REVIEWED_ASC, NHS_REVIEWED_DESC, NHS_VERIFIED_ASC, NHS_VERIFIED_DESC, INSERTED_AT_ASC, INSERTED_AT_DESC
```

File: packages/admin/e2e/__features__/legal_entities_list.feature [hunk 19]
```diff
@@ -11,22 +11,27 @@ Background: User is authorized and on DECLARATION_LIST_PAGE
...
 
 Scenario: Search for a Legal Entity using EDRPOU
   Given I fill in input search field with correct value
   And I submit form
-  Then I should see grid with name, edrpou, addresses, status, nhsVerified, insertedAt, details
-  And I should be able to order table by edrpou_ASC, edrpou_DESC, status_ASC, status_DESC, nhsVerified_ASC, nhsVerified_DESC, insertedAt_ASC, insertedAt_DESC
+  Then I should see grid with name, edrpou, addresses, status, nhsReview, nhsVerified, insertedAt, details
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_list.feature]:
```suggestion
  Then I should see table:
  | id  			  | name 	 | edrpou   | type 		  | addresses         | status | nhsReview | nhsVerified | insertedAt | action  |
  | 53XXX1d-XXXX-4fa3 | Lymych M | 31023234 | Description | Kyiv, Bazhana 12b | ACTIVE | true  	   | false 		 | 12.11.2018 | details |
  | 11XXX2X-XXXX-XXXX | Boris  M | 31323234 | Description | Kyiv, Bazhana 12b | ACTIVE | false 	   | true 		 | 12.11.2018 |	details |
```

File: packages/admin/e2e/__features__/legal_entities_list.feature [hunk 20]
```diff
@@ -11,22 +11,30 @@ Background: User is authorized and on DECLARATION_LIST_PAGE
...
   Given I chose value from drop down list 
-  Then I should see grid with name, edrpou, addresses, status, nhsVerified, insertedAt, details
-  And I should be able to order table by edrpou_ASC, edrpou_DESC, status_ASC, status_DESC, nhsVerified_ASC, nhsVerified_DESC, insertedAt_ASC, insertedAt_DESC
+  Then I should see table:
+  | id  			  | name 	 | edrpou   | type 		  | addresses         | status | nhsReview | nhsVerified | insertedAt | action  |
+  | 53XXX1d-XXXX-4fa3 | Lymych M | 31023234 | Description | Kyiv, Bazhana 12b | ACTIVE | true  	   | false 		 | 12.11.2018 | details |
+  | 11XXX2X-XXXX-XXXX | Boris  M | 31323234 | Description | Kyiv, Bazhana 12b | ACTIVE | false 	   | true 		 | 12.11.2018 |	details |
+  And I should be able to order table by edrpou_ASC, edrpou_DESC, status_ASC, status_DESC, nhsReviewed_ASC, nhsReviewed_DESC, nhsVerified_ASC, nhsVerified_DESC, insertedAt_ASC, insertedAt_DESC
 
 Scenario: Search for a Legal Entity using SETTLEMENT_NAME
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_list.feature]:
```suggestion
```

File: packages/admin/e2e/__features__/legal_entities_list.feature [hunk 21]
```diff
@@ -11,22 +11,30 @@ Background: User is authorized and on DECLARATION_LIST_PAGE
...
   Given I fill in SETTLEMENT_SEARCH field with correct value
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_list.feature]:
```suggestion
```

File: packages/admin/e2e/__features__/legal_entities_list.feature [hunk 22]
```diff
@@ -11,22 +11,30 @@ Background: User is authorized and on DECLARATION_LIST_PAGE
...
   And I submit form
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_list.feature]:
```suggestion
```

File: packages/admin/e2e/__features__/legal_entities_list.feature [hunk 23]
```diff
@@ -11,22 +11,30 @@ Background: User is authorized and on DECLARATION_LIST_PAGE
...
-  Then I should see grid with name, edrpou, addresses, status, nhsVerified, insertedAt, details
-  And I should be able to order table by edrpou_ASC, edrpou_DESC, status_ASC, status_DESC, nhsVerified_ASC, nhsVerified_DESC, insertedAt_ASC, insertedAt_DESC
+  Then I should see grid with name, edrpou, addresses, status, nhsReview, nhsVerified, insertedAt, details
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_list.feature]:
```suggestion
```

File: packages/admin/e2e/__features__/legal_entities_list.feature [hunk 24]
```diff
@@ -11,22 +11,30 @@ Background: User is authorized and on DECLARATION_LIST_PAGE
...
+  And I should be able to order table by edrpou_ASC, edrpou_DESC, status_ASC, status_DESC, nhsReviewed_ASC, nhsReviewed_DESC, nhsVerified_ASC, nhsVerified_DESC, insertedAt_ASC, insertedAt_DESC
```
Comment (burlachenkovv) [inline packages/admin/e2e/__features__/legal_entities_list.feature]:
```suggestion
```

**Pull request merged**